tools/update_idna_data.py

```diff
@@ -1,53 +0,0 @@
-#!/usr/bin/env python
```

Comment:
We don't have to delete these files in this stage, right? If so, I say lets keep them for reference, and delete when the consumer python scripts are gone.